libs/base/Control/Monad/RWS/CPS.idr

```diff
@@ -0,0 +1,133 @@
+||| Note: The difference to a 'stricht' RWST implementation is
+||| that accumulation of values does not happen in the
+||| Applicative and Monad instances but when invoking `Writer`-specific
+||| functions like `writer` or `listen`.
```

Comment:
typos: "stricht" -> "strict"

Reply:
Fixed, thanks. I'm Swiss. We are quite fond of our 'ch's. :-)